Options/Applicative/Help/Core.hs

```diff
@@ -59,12 +59,13 @@ optDesc pprefs style info opt =
         = mempty
         | isEmpty chunk || not (descSurround style)
         = mappend chunk suffix
-        | hinfoDefault info
-        = mappend (fmap brackets chunk) suffix
+        --  | hinfoDefault info
+        -- = mappend chunk suffix
         | null (drop 1 descs)
         = mappend chunk suffix
         | otherwise
-        = mappend (fmap parens chunk) suffix
```

Comment:
This set of parenthesis was important when there's more than one name

Reply:
```haskell
      render chunk
        | not show_opt
        = mempty
        | not (hinfoDefault info) && length descs > 1
        = fmap parens chunk <> suffix
        | otherwise
        = chunk <> suffix
```

Seems to do the trick

Reply:
Done